query.gql

```diff
@@ -3,11 +3,12 @@ module.exports = `query($repo: String!, $owner: String!, $branch: String!) {
     ref(qualifiedName: $branch) {
       target {
         ... on Commit {
-          deployments(last: 1) {
+          deployments(last: 100) {
```

Comment:
I'm not very comfortable with querying 100 deployments every time. Maybe add that as a configuration option?